lab_2_keywords_cooccurrence/main.py

```diff
@@ -48,13 +36,30 @@ def extract_candidate_keyword_phrases(phrases: Sequence[str], stop_words: Sequen
 
     In case of corrupt input arguments, None is returned
     """
-    if not type_check(phrases, list) or not type_check(stop_words, list):
+    if not (phrases and isinstance(phrases, list) and stop_words and isinstance(stop_words, list)):
```

Comment:
this condition is confusing. It will be better if you write using the 'or' operator: `not  phrases or not  isinstance(phrases, list) ...`